lib/VMwareWebService/broker_timeout.rb

```diff
@@ -0,0 +1,64 @@
+module Timeout
+  def timeout(sec, klass = nil)   #:yield: +sec+
+    return yield(sec) if sec == nil or sec.zero?
+    exception = klass || Class.new(ExitException)
+    state_lock = Mutex.new
+    state = :sleeping
+    toid = Class.new.object_id
```

Comment:
I don't think this is needed...I believe you had it before just for logging purposes.